src/test/pages/IsaacApp.test.tsx

```diff
@@ -139,4 +139,36 @@ describe("IsaacApp", () => {
         );
         expect(newsPodLinks.slice(featuredNewsPodLinks.length)).toEqual(nonFeaturedNewsPodLinks);
     });
+
+    it('should show the promo content banner for anonymous users', async () => {
+        renderTestEnvironment({role: "ANONYMOUS"});
+        await screen.findByTestId("main");
+        const promoContent = await screen.findByTestId("promo-content");
+        expect(promoContent).toBeInTheDocument();
+    })
+
+    let roles = ["STUDENT", "TUTOR", "TEACHER", "EVENT_LEADER", "EVENT_MANAGER", "CONTENT_EDITOR", "ADMIN"]
```

Comment:
`const roles: UserRole[]`. That way you can get rid of the type cast in `renderTestEnvironment`.

Reply:
Changed to 

``` let roles = [
      "STUDENT",
      "TUTOR",
      "TEACHER",
      "EVENT_LEADER",
      "EVENT_MANAGER",
      "CONTENT_EDITOR",
      "ADMIN",
    ] as (UserRole | "ANONYMOUS")[];
```

Due to line 178, "ANONYMOUS" is accepted into renderTestEnvironment, but it isn't a UserRole.